pyproject.toml

```diff
@@ -319,10 +319,12 @@ armi = [
     "testing/reactors/anl-afci-177/*",
     "testing/reactors/c5g7/*",
     "testing/reactors/godiva/*",
+    "testing/reactors/smallHexReactor/*",
+    "testing/resources/armiRun-SHUFFLES.txt",
+    "testing/resources/armiRun-SHUFFLES.yaml",
+    "testing/resources/COMPXS.ascii",
```

Comment:
Is there anything in testing/resources or testing/reactors that you would want to exclude? Could maybe simplify by including everything in the directories

Reply:
@john-science Actually I meant the opposite of what you did 😅 No worries -- it's not important. 

Like I was suggesting doing `testing/reactors/*` and `testing/resources/*` (if that's in fact the behavior you want)